rotkehlchen/tests/unit/globaldb/test_globaldb_consistency.py

```diff
@@ -371,3 +376,87 @@ def test_oracle_ids_in_asset_collections(globaldb: 'GlobalDBHandler'):
 
     if len(mismatches) > 0:
         pytest.fail('\n'.join(mismatches))
+
+
+def test_remote_updates_consistency_with_packaged_db(
+        tmpdir_factory: 'pytest.TempdirFactory',
+        messages_aggregator: 'MessagesAggregator',
+        database: 'DBHandler',
+):
+    """Test that the remote updates are consistent with the packaged db for:
+    - Location asset mappings
+    - Location unsupported assets"""
+    temp_data_dir = Path(tmpdir_factory.mktemp(GLOBALDIR_NAME))
+    (old_db_dir := temp_data_dir / GLOBALDIR_NAME).mkdir(parents=True, exist_ok=True)
+    request.urlretrieve(
+        url='https://github.com/rotki/rotki/raw/v1.33.0/rotkehlchen/data/global.db',
```

Comment:
here you have hardcoded the version. Shouldn't this be dinamic? or it doesn't matter because it will update it

Reply:
this version is hardcoded because the `location_asset_mappings` and `location_unsupported_assets` tables were added from this version and after this they will be updated